internal/distro/fedora31/distro.go

```diff
@@ -646,7 +645,7 @@ func qemuAssembler(format string, filename string, uefi bool, size uint64) *osbu
 				{
 					Start: 2048,
 					Size:  972800,
-					Type:  &fstype,
```

Comment:
Will this pass the tests? It is lower-case and in the same for f32 it is upper-case?

Reply:
Actually unit tests were unhappy when it was the same in f31 and f32. It was either this change or fix in test definition.